src/main/java/cz/muni/fi/pa165/dao/TicketDao.java

```diff
@@ -0,0 +1,23 @@
+package cz.muni.fi.pa165.dao;
+
+import cz.muni.fi.pa165.entity.*;
+
+import java.time.LocalDate;
+import java.util.List;
+
+/**
+ * Interface of DAO for Ticket entity
+ * @author Tomas Rudolf
+ */
+public interface TicketDao {
+    public void create(Ticket c);
+    public Ticket findById(Long id);
+    public List<Ticket> findAll();
+    public List<Ticket> findByCreationDate(LocalDate creationDate);
+    public List<Ticket> findByUpdateDate(LocalDate updateDate);
+    // FIXME Tomas milestone1 - Uncomment after classes are in repository
+    /*public List<Ticket> findByPerformance(Performance performance);
+    public List<Ticket> findByUser(User user);*/
+    public void update(Ticket c);
```

Comment:
Vzhledem k tomu, že implementace používá merge, kterážto vrací novou instanci, neměla by ji vracet i update? Pokud by na ní pak nikdo už nechtěl volad setXXX, tak to asi nevadí, t. j. by se to muselo volat vždycky nakonci, aby se to nerozbilo.

Reply:
S volanim na konci jsem prave pocital. Ja nevim kdy zanikne persistance, jelikoz je tam dodana pres anotaci. Tohle pocita s tim, ze vytvorime objekt, dame DAO.create(objekt) a pak v budoucnu findByNeco, upravime a dame update toho objektu

Reply:
Teda takhle :) Cekal jsem, ze mi to nekdo pri testech vrati, ze takhle to opravdu nejde a nebo zjistime, ze to tak ma fungovat